build/source-go.txt

```diff
@@ -0,0 +1,8 @@
+## please update this source info whenever new cri-api content is used 
```

Comment:
boiler plate is missing.
Thanks a lot for adding this file to track which API the runtime is syncing up with.

Reply:
will add the license header section